package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "watch": "nodemon --watch ./ --watch views --watch controllers --watch build --watch helpers --ext js,hbs,json ./bin/www",
-    "debug": "nodemon --inspect=9310 --watch controllers --watch helpers --watch views --watch helpers --ext js,hbs,json ./bin/www",
-    "debug:container": "nodemon --inspect=0.0.0.0:9310 --watch controllers --watch helpers --watch views --watch helpers --ext js,hbs,json ./bin/www",
+    "watch": "nodemon --config nodemon.json",
+    "debug": "nodemon --inspect=9310 --config nodemon.json",
+    "debug:container": "nodemon --inspect=0.0.0.0:9310 --config nodemon.json",
```

Comment:
i like it 🕺

Reply:
you could even remove the --config param (because nodemon.json is default) but it is easier to understand for everyone if it is explicit.